README.md

```diff
@@ -1,11 +1,22 @@
-UCL API Main Repository
+# UCL API [![Build Status](https://travis-ci.org/uclapi/uclapi.svg?branch=master)](https://travis-ci.org/uclapi/uclapi) [![codecov](https://codecov.io/gh/uclapi/uclapi/branch/master/graph/badge.svg)](https://codecov.io/gh/uclapi/uclapi)
+
+## What is the UCL API?
+UCL API is a **student-built** platform for **student developers** to improve the **student experience** of everyone at UCL.
+
+### Our Goal
+Create a ridiculously simple, documentation first, and comprehensive API around UCL's digital services and establish an ecosystem of third party UCL apps and services that use the API.
+
+### Interested in using it?
+Find out more at our [website](https://uclapi.com)
+
+### Interested in helping build it?
+Read on more to find out how to setup and build the API. From here you can start writing your own changes and submitting them, read our [Contribution Guide](CONTRIBUTING.md) to learn more about how you can contribute.
 
 ## Setting Up and Building
 This will walk you through setting yourself up a local development environment. This is different to if you want to deploy this into production, but luckily we have an internal set of installation scripts for this. Therefore, this guide will not walk you through, for example, getting Gunicorn or Nginx running. This is for simply getting an environment up that can be used to develop the API. Testing with Nginx and Gunicorn is done in staging prior to a production deployment.
 
 ### Requirements
-We only support development under Linux. macOS is unofficially supported, and we have some instructions below on how to get the Oracle Instant Client working on macOS. All Windows developers can and should use Bash on Ubuntu on Windows under Windows 10 (Creators Update) for testing, as the environment is Ubuntu 16.04.2 LTS; this is exactly the distribution and version of Linux that we use on both our Staging and Production web servers.
+We only support development under Linux. macOS is unofficially supported, and we have some instructions below on how to get the Oracle Instant Client working on macOS. All Windows developers can and should use Bash on Ubuntu on Windows under Windows 10 for testing, as the environment is Ubuntu 18.04.1 LTS; this is exactly the distribution and version of Linux that we use on both our Staging and Production web servers. Make sure you are installing Ubuntu (This defaults to the most recent LTS release) or Ubuntu 18.04 from the window store, although you should have no issues with the 16.04 version we cannot guarantee this.
 
 Note that since the Creators Update (which includes 16.04.2; if you have not upgraded from Ubuntu 14 then there are tutorials online to do this) we have experienced zero issues building and running under Bash on Ubuntu on Windows.
```

Comment:
```suggestion
Note that since the Creators Update of Windows 10, we have not had any issues building or running UCL API on Windows 10 under the Linux Subsystem. However, we always recommend that Windows developers stay up to date with the latest feature updates to Windows (released twice a year) as Microsoft is rapidly developing and improving WSL.
```